test/tsconfig.json

```diff
@@ -142,4 +143,4 @@
         "rule-tester/testData.ts",
         "rule-tester/utilsTests.ts"
     ]
-}
+}
```

Comment:
Sure. However, this was added automatically by the grunt build. So you may want to change the build somehow. I am on Windows using git-bash. My .git/config file has: 

    core.autocrlf=input
    core.eol=lf